fix(cloudflare): dynamic-imports-analyzer (#238)
withastro/adapters: .changeset/sweet-avocados-trade.md

---
'@astrojs/cloudflare': patch
---

> Fixes an issue, where `astro build` fails if it can't clean up dynamic imports.

Author: ```suggestion
Fixes an issue, where `astro build` throws an error if it can't clean up dynamic imports for prerendered pages.
```

> Fixes an issue, where `astro build` throws an error if it can't clean up dynamic imports for prerendered pages.

Author: ```suggestion
Fixes `astro build` which throws an error if it can't clean up dynamic imports for prerendered pages.
```
